pkg/api/rest/resttest/resttest.go

```diff
@@ -133,8 +133,11 @@ func (t *Tester) TestDelete(createFn func() runtime.Object, wasGracefulFn func()
 // Test graceful deletion.
 // TODO(wojtek-t): Change it to use AssignFunc instead.
 func (t *Tester) TestDeleteGraceful(createFn func() runtime.Object, expectedGrace int64, wasGracefulFn func() bool) {
-	t.testDeleteGracefulHasDefault(createFn(), expectedGrace, wasGracefulFn)
-	t.testDeleteGracefulUsesZeroOnNil(createFn(), 0)
+	t.TestDeleteGracefulHasDefault(createFn(), expectedGrace, wasGracefulFn)
```

Comment:
It was to support specific in depth testing of strategies with parameters.  Will comment